website_variants_extra/static/src/less/website.less

```diff
@@ -0,0 +1,32 @@
+.website-sale-actions{
+    .fa {
+        font-size: 30px;
+        color: black;
+    }
+    .sale-action{
+        font-size: 14px;
+        padding-left: 5px;
+        color: black;
+    }
+    margin-top: 10px;
+}
+.actions_column{
+    padding-left: 25px;
+    padding-bottom: 20px;
+}
+.small-text .text-green{
+    color: @link-color;
+}
+.actions_column{
+    background-color:#ecf0f1;
+}
+#add_to_cart{
+    background-color:#f39c12;
+}
+#product_with_variants{
+    color: @link-color;
+}
+.js_quantity.form-control {
+    background-color: white;
+
+}
```

Comment:
please put all these classes namespaced, do not leave them just as simple CSS rules, this mean you have to nest them under a parent selector.